frontend/src/lib/components/PropertyKeyInfo.scss

```diff
@@ -5,6 +5,7 @@
     align-items: center;
     vertical-align: middle;
     overflow: hidden;
+    max-width: 100%;
 
     .PropertyKeyInfo__text {
         max-width: 400;
```

Comment:
Unrelated to this PR but I think this is a typo.

Reply:
Good catch, that might have been me some other time 🤦  I'm pretty sure it is doing nothing so I'll remove it